docs/src/howto/networking/default-dns.md

```diff
@@ -0,0 +1,86 @@
+# How to use default DNS
+
+Canonical Kubernetes includes a default DNS (Domain Name System) essential for internal communication within your cluster. 
+When enabled, the DNS facilitates service discovery
+by assigning each service a DNS name. 
+
+## What you'll need
+
+This guide assumes the following:
+
+- You are installing on Ubuntu 22.04 or later, **or** another OS which supports
+  snap packages (see [snapd support])
+- You have root or sudo access to the machine
+- You have an internet connection
```

Comment:
Should we really start _that_ low? I assume the reader has a technical background.
Also, IIRC how-to sections are for people that already know k8s and what they are doing. So we do not need to start from scratch (correct me if I'm wrong @evilnick ).